templates/CRM/Contribute/Form/Contribution/Confirm.tpl

```diff
@@ -100,9 +100,17 @@
                 {else}
 {crmRegion name="contribution-confirm-recur"}
                     {if $installments}
-                        <p><strong>{ts 1=$frequency_interval 2=$frequency_unit 3=$installments}I want to contribute this amount every %1 %2(s) for %3 installments.{/ts}</strong></p>
+                      {if $frequency_interval > 1}
+                        <p><strong>{ts 1=$frequency_interval 2=$frequency_unit 3=$installments}I want to contribute this amount every %1 %2s for %3 installments.{/ts}</strong></p>
+                      {else}
+                        <p><strong>{ts 1=$frequency_interval 2=$frequency_unit}I want to contribute this amount every %1 %2s.{/ts}</strong></p>
```

Comment:
This line is missing $installments. Looks like this line and line 110 are swapped.
We should be using the plural ts syntax instead of '%2s'
{ts count=$count plural='Found %count contacts'}Found %count contact{/ts}